server2/Arcadia.Assistant/Arcadia.Assistant.DeviceRegistry/DeviceRegistry.cs

```diff
@@ -0,0 +1,192 @@
+namespace Arcadia.Assistant.DeviceRegistry
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Fabric;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using Castle.Core.Internal;
+
+    using Contracts;
+    using Contracts.Models;
+
+    using Microsoft.Extensions.Logging;
+    using Microsoft.ServiceFabric.Data.Collections;
+    using Microsoft.ServiceFabric.Services.Communication.Runtime;
+    using Microsoft.ServiceFabric.Services.Runtime;
+
+    using Models;
+
+    /// <summary>
+    ///     An instance of this class is created for each service replica by the Service Fabric runtime.
+    /// </summary>
+    internal sealed class DeviceRegistry : StatefulService, IDeviceRegistry
+    {
+        private const string DeviceRegistryKey = "device_tokens";
+        private const string DeviceOwnersRegistryKey = "device_employee";
+        private readonly ILogger logger;
+        private readonly int OperationTimeout = 5;
+
+        public DeviceRegistry(StatefulServiceContext context, ILogger logger)
+            : base(context)
+        {
+            this.Timeout = TimeSpan.FromMinutes(this.OperationTimeout);
+            this.logger = logger;
+        }
+
+        private TimeSpan Timeout { get; }
```

Comment:
why? Just do it in a field initializer, you don't need two fields for it

Reply:
Done